### ISSUE TITLE
Feature request : Play all tracks

### ISSUE THREAD

**Issue by qkdreyer** [jaedb/Iris#250]
![image](https://user-images.githubusercontent.com/717869/34933504-65fb78b6-f9d7-11e7-91b6-740e19bd96ac.png)

Official Spotify.app has a "Play all tracks" feature.
Sadly, I didn't manage to find the same feature in Iris :(
I wouldn't mind implementing such a thing, all I need would be a starting point :)

**Comment by jaedb** [comment]
Released to production in 3.12.0

**Comment by qkdreyer** [comment]
Excuse me, but I didn't manage to find a "Play all" button for "/iris/#/library/tracks" URI

**Comment by jaedb** [comment]
Hmm, no you're absolutely right. I believe this might have got tangled up in the work for _Play all_ in search results.

**Comment by qkdreyer** [comment]
Any updates ?

**Comment by jaedb** [comment]
I've got this running locally, but you'll find it's a bit cumbersome. Spotify's track library consists of not only tracks you've added, but also the tracks of every album you've got in your library as well. This means your track library is likely to be huge.

As a result, it will take a long time to load all the library tracks and then add each one to the queue.

I've got the functionality available on the develop branch, and will be in the next release. Then I can gauge feedback on whether it needs a different approach.
